src/containers/Review/ReviewSectionAssignment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Segment } from 'semantic-ui-react'
+import { NoMatch, Loading } from '../../components'
+import useGetFullReviewStructure from '../../utils/hooks/useGetFullReviewStructure'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+
+// Component renders a line for review assignment within a section
+// to be used in ReviewHome and Expansions
+const ReviewSectionAssignment: React.FC<{
+  sectionId: number
+  assignment: AssignmentDetailsNEW
+  fullApplicationStructure: FullStructure
+}> = ({ sectionId, assignment, fullApplicationStructure }) => {
+  const { fullReviewStructure, error } = useGetFullReviewStructure({
+    reviewAssignmentId: assignment.id,
+    fullApplicationStructure,
+    filteredSectionIds: [sectionId],
+  })
+
+  if (error) return <NoMatch />
```

Comment:
I was considering error to be something unexpected, and in this case show an error message. I don't  see yet what would be this case, but I wonder how would this show for a section that isn't assigned to the current user? No errors, right?

Reply:
it's error from the query or if reviewAssignment does not exist (regardless of the use as useGerReviewNewQuery doesn't filter by user), should't return not match thought good point (also found another place where error should be displayed): 717f0e77b4ad3e88ccd4d8ff03af62cac65ee524